lib/maths/unittest/CQuantileSketchTest.cc

```diff
@@ -51,6 +53,8 @@ void testSketch(maths::CQuantileSketch::EInterpolation interpolation,
         double q = static_cast<double>(i) / 20.0;
         double xq = samples[static_cast<std::size_t>(static_cast<double>(N) * q)];
         double sq;
+        BOOST_REQUIRE_EQUAL(sketch.quantile(100.0 * q, sq),
+                            fastSketch.quantile(100.0 * q, sq));
```

Comment:
Another test (potentially `disabled` by default) benchmarking `CFastQuantileSketch` vs `CQuantileSketch` would be "nice to have"

Reply:
We do have a couple of tests like this, but I'm not 100% sold on them. I don't think we could enable such a test because of CI running in virtual environment. As such I don't think this would ever get run, except if someone to sees a performance degradation in the QA benchmarks. In such cases changes to this class would usually be a clear candidate (or not) from the commit list and it is really easy to write the code to check this: just add `for (std::size_t i = 0; i < 10000; ++i)` around one of the calls to `testSketch` and time before and after. As a result, I don't see such a test adding value on top of performance monitoring QA does. WDYT?

Reply:
Yep, that's good enough for me Tom.